reference/catalog/custom-fields.v3.yml

```diff
@@ -0,0 +1,479 @@
+openapi: '3.0.3'
```

Comment:
@olysenko what's the rationale for removing Catalog Product Custom Fields from the Catalog Products file? Let's discuss.